RedHat-UX/red-hat-design-system
# docs(tile): update tile hover state text and images

## What I did

1. Added a sentence about the arrow animation under the Hover section and updated both hover state images
2. Noted that the arrow doesn't move in focus state

## Testing Instructions

1. Check Tile's Style page in [DP](https://deploy-preview-1290--red-hat-design-system.netlify.app/elements/tile/style/#hover)
